Lib/fontTools/varLib/varStore.py

```diff
@@ -184,6 +184,71 @@ def interpolateFromDeltas(self, varDataIndex, deltas):
 		return self.interpolateFromDeltasAndScalars(deltas, scalars)
 
 
+class VarDataItem(object):
```

Comment:
I don't think this belongs here.

Reply:
yeah, maybe should be moved to `fontTools.varLib.cff`